internal/reconciliations/proxy/sidecars_test.go

```diff
@@ -141,3 +141,7 @@ func (s StatusMock) SetError(_ context.Context, _ described_errors.DescribedErro
 func (s StatusMock) SetDeleting(_ context.Context, _ client.Client, _ *operatorv1alpha1.Istio, _ metav1.Condition, _ ...time.Duration) (ctrl.Result, error) {
 	return s.result, s.err
 }
+
```

Comment:
Maybe the name for `described_errors.DescribedError` should be adjusted (e.g. `described_errors.DescribedState`)?